### PR TITLE
updated label_quality_utils.py and rebuilt the doc

### PR DESCRIPTION
**Pull request by ethanotran**
Updated label_quality_utils.html after PR: 
Screenshot (687)

### CODE REVIEW

File: cleanlab/internal/label_quality_utils.py [hunk 1]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
 
     Parameters
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:97]: These changes do not match the comments I left on this file in your other PR here:
https://github.com/cleanlab/cleanlab/pull/473/files#diff-01ba518873fbd9bb46ff9e13a2af9adc756c202b723dadfa885de9a5327b81f8

You'll want to carefully look through those.

File: cleanlab/internal/label_quality_utils.py [hunk 2]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
...
     ----------
-    pred_probs : np.ndarray (shape (N, K))
+    pred_probs : np.ndarray
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:99]:
```suggestion
    pred_probs:
```

File: cleanlab/internal/label_quality_utils.py [hunk 3]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
...
       Each row of this matrix corresponds to an example x and contains the model-predicted
       probabilities that x belongs to each possible class: P(label=k|x)
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob :
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:103]:
```suggestion
    min_allowed_prob: 
```

File: cleanlab/internal/label_quality_utils.py [hunk 4]
```diff
@@ -96,17 +96,17 @@ def get_normalized_entropy(pred_probs: np.ndarray, min_allowed_prob: float = 1e-
...
 
-    min_allowed_prob : float, default=1e-6
+    min_allowed_prob : 
       Minimum allowed probability value. Entries of `pred_probs` below this value will be clipped to this value.
       Ensures entropy remains well-behaved even when `pred_probs` contains zeros.
 
     Returns
     -------
-    entropy : np.ndarray (float)
+    entropy : np.ndarray
```
Comment (jwmueller) [inline cleanlab/internal/label_quality_utils.py:109]:
```suggestion
    entropy:
```

File: cleanlab/internal/label_quality_utils.py (hunk 1 above)
Comment (jwmueller) [reply cleanlab/internal/label_quality_utils.py:97]: I have made the necessary changes for you as "suggestions" so all you need to do now is commit each of these suggestions. But in the future, you want to be more precise when addressing comments our team makes! 
Remember all the little details really matter for high-quality code

Comment (ethanotran) [reply cleanlab/internal/label_quality_utils.py:97]: Whoops, my bad. Fixed it.

Comment (jwmueller) [reply cleanlab/internal/label_quality_utils.py:97]: thanks this PR is good to go!

**Review by jwmueller — APPROVED** [review]
Thanks for the contribution @ethanotran 
You can now open a separate PR to make these updates for most other source code files in our codebase (please include before/after screenshots like you nicely did this time!).

It will take some time for us to merge this PR since we are in the midst of resolving some CI issues.

**Pull request merged**